tests/autopilot_pool_expand/aut_pool_expand_test.go

```diff
@@ -0,0 +1,215 @@
+package tests
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	torpedo_api "github.com/portworx/torpedo/drivers/api"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	. "github.com/portworx/torpedo/tests"
+)
+
+var (
+	testNameSuite            = "AutPoolResize"
+	timeout                  = 30 * time.Minute
+	scaleTimeout             = 2 * time.Hour
+	workloadTimeout          = 5 * time.Hour
+	retryInterval            = 30 * time.Second
+	unscheduledResizeTimeout = 10 * time.Minute
+)
+
+var ruleResizeBy50IfPoolUsageMoreThan50AddDisk = torpedo_api.AutopilotRule{
+	Name: "storage-pool-add-disk",
+	ConditionExpressions: []torpedo_api.AutopilotRuleCondition{
+		{
+			Key:      torpedo_api.PxPoolAvailableCapacityMetric,
+			Operator: torpedo_api.LabelSelectorOpLt,
+			Values:   []string{"80"},
+		},
+	},
+	Actions: []torpedo_api.AutopilotRuleAction{
+		{
+			Name: torpedo_api.StorageSpecAction,
+			Params: map[string]string{
+				torpedo_api.RuleActionsScalePercentage: "50",
+				torpedo_api.RuleScaleType:              torpedo_api.RuleScaleTypeAddDisk,
+				// torpedo_api.RuleMaxSize:                "322122547200",
+			},
+		},
+	},
+	Spec: torpedo_api.AutopilotSpec{
+		MatchLabels: map[string]string{"name": "storage-pool-resize"},
+	},
+}
+
+var ruleResizeBy50IfPoolUsageMoreThan50ResizeDisk = torpedo_api.AutopilotRule{
+	Name: "storage-pool-resize-disk",
+	ConditionExpressions: []torpedo_api.AutopilotRuleCondition{
+		{
+			Key:      torpedo_api.PxPoolAvailableCapacityMetric,
+			Operator: torpedo_api.LabelSelectorOpLt,
+			Values:   []string{"85"},
```

Comment:
value `85` does not match with rule name `ruleResizeBy50IfPoolUsageMoreThan50ResizeDisk`